CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 2.0.0 (2016-05-28)
```

Comment:
We’ll take care of the release on our end. Can you remove the changes to this file and maintain the `NEWS.md` filename?